cloudevents/go/cde/README.md

```diff
@@ -0,0 +1,101 @@
+# CDE - CloudEvents for Continuous Delivery
+
+Simple CLI and library to emit Cloud Events related to Continuous Delivery.
+
+The idea behind this framework and command-line interface is to empower you  emitting events related to a Continuous Delivery to enable interoperability between different tools, measure performance and gain visibility of your CD processes.
+
+
+## Usage
+
+Download the binary `cde` or build from source, by clonning this repository and run `make build`
+
+Set `CDE_SINK` environment variable to define where CloudEvents will be sent. You can do this by running:
+
+`export CDE_SINK=http://my-cloudevent-broker` 
+
+You can use [SockEye](https://github.com/n3wscott/sockeye) to visualize the cloud events.
+
+Integrate with your existing pipelines, repositories, and environments.
+
+## Supported Events
+
+Use `cde --help`
+
+The following events are supported:
+- [Environment Events](#environment-events)
+- [Service Events](#service-events)
+- [PipelineRun Events](#pipelinerun-events)
+- [TaskRun Events](#taskrun-events)
+- [Repository Events](#repository-events)
+            
+
+# CDE Events
+
+The following events are currently supported: 
+
+## Environment Events
+- Properties
+    - Id
+    - Name
+    - URL
+      
+Examples: 
+
+- **cd.environment.created.v1** `./cde env created --id staging --name "Staging Environment" --repo http://github.com/user/myrepo --data event=data`
```

Comment:
I need to make a poll to collect name ideas for the protocol - so we can agree on what we should use in the events.

**ab.cde.environment.created.v1** would be cool or **dev.cde.environment.created.v1**

Reply:
@afrittoli I do agree.. but I would hold the PR until we made that decision

Reply:
ok - we'll discuss the results of the poll and the name in the meeting today 07/06